app/tests/test_assets_endpoint.py

```diff
@@ -585,25 +592,27 @@ def test_asset_endpoint_patch_rename_asset(self):
         response = self.client.patch(
             path, data=changed_asset.get_json('patch'), content_type="application/json"
         )
-        json_data = response.json()
-        self.assertStatusCode(200, response)
-        self.assertEqual(changed_asset.json['id'], json_data['id'])
-        self.check_stac_asset(
-            changed_asset.json, json_data, collection_name, item_name, ignore=['item']
-        )
+        self.assertStatusCode(400, response)
```

Comment:
Same as above also test the response description to make sure that the 400 is due to a  `rename not allowed`